components/brave_wallet/common/BUILD.gn

```diff
@@ -78,7 +77,10 @@ static_library("common") {
     "//url",
   ]
 
-  public_deps = [ "//base" ]
+  public_deps = [
+    "//base",
+    "//brave/third_party/argon2",
+  ]
```

Comment:
this is not a public dependency.
public_dep is for dependencies which are coming from including public headers